zone/guild.cpp

```diff
@@ -234,8 +246,11 @@ void Client::RefreshGuildInfo()
 
 	guildrank = info.rank;
 	guild_id = info.guild_id;
-	GuildBanker = info.banker || guild_mgr.IsGuildLeader(GuildID(), CharacterID());
-
+	GuildBanker = info.banker ||
```

Comment:
What is happening here ?

Reply:
This could is my attempt for backwards compatibility with TI/UF.  These clients have a banker flag, while RoF uses the permissions system.  Therefore, I have set the GuildBanker Flag to be on, if the flag is set, the player is the guild leader, a guild leader using the TI/UF client sets the banker flag, or if the Client is an RoF client.

Then, in the banker system, it checks for the flag and the the RoF permission.